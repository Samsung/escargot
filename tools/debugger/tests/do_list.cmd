b do_list.js:20
b do_list.js:21
list
quit
