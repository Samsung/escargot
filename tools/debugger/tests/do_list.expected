Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_list.js:20
(escargot-debugger) b do_list.js:20
Breakpoint 1 at tools/debugger/tests/do_list.js:20
(escargot-debugger) b do_list.js:21
Breakpoint 2 at tools/debugger/tests/do_list.js:21
(escargot-debugger) list
=== Active breakpoints ===
 1: tools/debugger/tests/do_list.js:20
 2: tools/debugger/tests/do_list.js:21
(escargot-debugger) quit
